lib/voyager_helpers/liberator.rb

```diff
@@ -796,11 +796,20 @@ def merge_holding_item_into_bib(bib, holding, item, recap=false, conn=nil)
             h[key]['subfields'].insert(2, {"h"=>combined_call_no})
             record_hash['fields'] << h
           end
+          item_enum_chron = ''
+          if item[:enum]
+            item_enum_chron << "#{item[:enum]} "
+          end
+          if item[:chron]
+            item_enum_chron << "(#{item[:chron]})"
+          end
+          item_enum_chron.gsub!(/^(.*)[ ]+$/, '\1')
```

Comment:
I think it might be preferable use the logic on [these lines](https://github.com/pulibrary/voyager_helpers/blob/master/lib/voyager_helpers/liberator.rb#L252-L257) for getting the enum/chron string. 

Also can you add a spec for when both enum and chron are nil, as well as when one of them is nil? I think that when there is just the chron present that we wouldn't want the value to be surrounded by parentheses.